paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -473,10 +473,10 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
       } else {
         conv_pd = ConvFwdPrimitiveDesc(
-            src_md, weights_md, dst_md, strides, paddings, mkldnn_engine,
-            fuse_relu || fuse_brelu /*fuse_relu*/, fuse_residual_conn,
-            false /*fuse_brelu*/, fuse_brelu_threshold, output_shift_scale,
-            sum_scale, is_test);
+            src_md, weights_md, boost::none, dst_md, strides, paddings,
```

Comment:
What about defining the `bias_md = boost::none` before `if (bias) {...}` block, and calling the `ConvFwdPrimitiveDesc()` in a single place?

Reply:
> What about defining the `bias_md = boost::none` before `if (bias) {...}` block, and calling the `ConvFwdPrimitiveDesc()` in a single place?

After trying `boost::optional<memory::desc> bias_md{none_t}` and then call in single place. We found we could not pass reference as parameter. As we discussed it may copy big memory.
Later we decided to use `memory::desc* bias_md_p = nullptr;`